### DISCUSSION TITLE
commits in the repository from non-collaborating accounts

### DISCUSSION THREAD

**Discussion by CesarObedFL** [community/community#67145] [Other Feature Feedback, Questions, & Ideas]
### collaborators commits

Bug

### Body

I found an error with some commits that I uploaded to a repository with an account that is not a collaborator...
explanation: on my local machine I have two git accounts, one from my organization and another personal, I cloned a project in which I collaborate with my personal account and when I pushed the commit, it was uploaded with my credentials, username and email organization account, with which I am not a collaborator of the repository, my main local git account is that of the organization

**Comment by hafiz-muhammad** [comment]
To associate the correct email address with the commits, you need to modify the email of the commit to match the email of the account you want to use as the commit author, which can be done using this script: [https://gist.github.com/hafiz-muhammad/464da0a1783bd553c03d3ac422527987](https://gist.github.com/hafiz-muhammad/464da0a1783bd553c03d3ac422527987)
